tracer/src/Datadog.Trace/Util/Http/QueryStringObfuscation/Obfuscator.cs

```diff
@@ -3,9 +3,17 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
+// turns out strict formatting and optional compilation don't like each other
+#pragma warning disable SA1001, SA1116, SA1118
+
 using System;
+#if !NETCOREAPP3_1_OR_GREATER
 using System.Text.RegularExpressions;
```

Comment:
for my own understanding, are there compatibility reasons why we dont support below netcoreapp3.1?

Reply:
Yes, compiling for .NET 4.6.1 and .NET standard can be done, but requires reference to System.Memory & System.Reflection.Emit.Lightweight  (for DynamicMethod and System.Span<T>). Adding these references seems to causes issues with unit and integration tests because of conflict, so presumably that would happen in real world apps too.

Reply:
good to know thanks!